mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -56,6 +62,16 @@ CLASS none/se net/minecraft/entity/Entity
 		ARG 1 commandName
 	METHOD a setUuid (Ljava/util/UUID;)V
 		ARG 0 uuid
+	METHOD a dropItems (Lnone/aeo;I)Lnone/yy;
```

Comment:
`dropStacks` is also the name of the static helper function in Block, though the methods that actually drops items upon the block being broken is called `dropItems` and I think it should stay that way..?